### ISSUE TITLE
 Error: Could not locate the bindings file.

### ISSUE THREAD

**Issue by 1Dilva** [PetyXbron/minecraft-bot#19]
Hey, running this on a Pebblehost service. The bot worked ONCE then it never worked again I reset files deleted everything reinstalled no hope I was hoping to get some help here. Console vvvv I also contacted them and they said "we don't support SQLite" but that makes no sense because the bot worked once and there should be a fix around it. I hope you are still active here and I hope I get some help thanks

_____________________________
01.02 09:36:01 [Bot] /Bot3/node_modules/bindings/bindings.js:135
01.02 09:36:01 [Bot] throw err;
01.02 09:36:01 [Bot] ^
01.02 09:36:01 [Bot] Error: Could not locate the bindings file. Tried:
01.02 09:36:01 [Bot] → /Bot3/node_modules/better-sqlite3/build/better_sqlite3.node
01.02 09:36:01 [Bot] → /Bot3/node_modules/better-sqlite3/build/Debug/better_sqlite3.node
01.02 09:36:01 [Bot] → /Bot3/node_modules/better-sqlite3/build/Release/better_sqlite3.node
01.02 09:36:01 [Bot] → /Bot3/node_modules/better-sqlite3/out/Debug/better_sqlite3.node
01.02 09:36:01 [Bot] → /Bot3/node_modules/better-sqlite3/Debug/better_sqlite3.node
01.02 09:36:01 [Bot] → /Bot3/node_modules/better-sqlite3/out/Release/better_sqlite3.node
01.02 09:36:01 [Bot] → /Bot3/node_modules/better-sqlite3/Release/better_sqlite3.node
01.02 09:36:01 [Bot] → /Bot3/node_modules/better-sqlite3/build/default/better_sqlite3.node
01.02 09:36:01 [Bot] → /Bot3/node_modules/better-sqlite3/compiled/16.17.0/linux/x64/better_sqlite3.node
01.02 09:36:01 [Bot] → /Bot3/node_modules/better-sqlite3/addon-build/release/install-root/better_sqlite3.node
01.02 09:36:01 [Bot] → /Bot3/node_modules/better-sqlite3/addon-build/debug/install-root/better_sqlite3.node
01.02 09:36:01 [Bot] → /Bot3/node_modules/better-sqlite3/addon-build/default/install-root/better_sqlite3.node
01.02 09:36:01 [Bot] → /Bot3/node_modules/better-sqlite3/lib/binding/node-v93-linux-x64/better_sqlite3.node
01.02 09:36:01 [Bot] at bindings (/Bot3/node_modules/bindings/bindings.js:126:9)
01.02 09:36:01 [Bot] at new Database (/Bot3/node_modules/better-sqlite3/lib/database.js:48:64)
01.02 09:36:01 [Bot] at Object.<anonymous> (/Bot3/node_modules/quick.db/src/index.js:7:15)
01.02 09:36:01 [Bot] at Module._compile (node:internal/modules/cjs/loader:1126:14)
01.02 09:36:01 [Bot] at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
01.02 09:36:01 [Bot] at Module.load (node:internal/modules/cjs/loader:1004:32)
01.02 09:36:01 [Bot] at Function.Module._load (node:internal/modules/cjs/loader:839:12)
01.02 09:36:01 [Bot] at Module.require (node:internal/modules/cjs/loader:1028:19)
01.02 09:36:01 [Bot] at require (node:internal/modules/cjs/helpers:102:18)
01.02 09:36:01 [Bot] at Object.<anonymous> (/Bot3/node_modules/quick.db/index.js:7:18) {
01.02 09:36:01 [Bot] tries: [
01.02 09:36:01 [Bot] '/Bot3/node_modules/better-sqlite3/build/better_sqlite3.node',
01.02 09:36:01 [Bot] '/Bot3/node_modules/better-sqlite3/build/Debug/better_sqlite3.node',
01.02 09:36:01 [Bot] '/Bot3/node_modules/better-sqlite3/build/Release/better_sqlite3.node',
01.02 09:36:01 [Bot] '/Bot3/node_modules/better-sqlite3/out/Debug/better_sqlite3.node',
01.02 09:36:01 [Bot] '/Bot3/node_modules/better-sqlite3/Debug/better_sqlite3.node',
01.02 09:36:01 [Bot] '/Bot3/node_modules/better-sqlite3/out/Release/better_sqlite3.node',
01.02 09:36:01 [Bot] '/Bot3/node_modules/better-sqlite3/Release/better_sqlite3.node',
01.02 09:36:01 [Bot] '/Bot3/node_modules/better-sqlite3/build/default/better_sqlite3.node',
01.02 09:36:01 [Bot] '/Bot3/node_modules/better-sqlite3/compiled/16.17.0/linux/x64/better_sqlite3.node',
01.02 09:36:01 [Bot] '/Bot3/node_modules/better-sqlite3/addon-build/release/install-root/better_sqlite3.node',
01.02 09:36:01 [Bot] '/Bot3/node_modules/better-sqlite3/addon-build/debug/install-root/better_sqlite3.node',
01.02 09:36:01 [Bot] '/Bot3/node_modules/better-sqlite3/addon-build/default/install-root/better_sqlite3.node',
01.02 09:36:01 [Bot] '/Bot3/node_modules/better-sqlite3/lib/binding/node-v93-linux-x64/better_sqlite3.node'
01.02 09:36:01 [Bot] ]
01.02 09:36:01 [Bot] }
01.02 09:36:01 [PebbleHost Loader] File ./Bot3/index.js stopped

**Comment by PetyXbron** [comment]
Check yours quick.db version with the command `npm list quick.db`.
Downgrading [quick.db](https://www.npmjs.com/package/quick.db) to [7.1.3](https://www.npmjs.com/package/quick.db/v/7.1.3) should fix the issue.
You can use this console command when the process is off: **`npm i quick.db@7.1.3 --save --save-exact`**

**Comment by PetyXbron** [comment]
Also, did you run the command `npm install` before running the project?

**Comment by 1Dilva** [comment]
The quick.db version is already 7.1.3 and yes I have run the npm install command

**Comment by PetyXbron** [comment]
Yeah, it worked once because the first run of the bot does not require SQLite, then it needs.
SQLite is a solution for saving the message ID of statusCH. As the Console vvvv (Pebblehost service?) support said, they are not supporting SQLite and that, unfortunately, you cannot use the statusCH feature.
I will try to find another solution for saving the message ID of statusCH.

**Comment by PetyXbron** [comment]
Also, I would suggest that you consider using another hosting service.

**Comment by 1Dilva** [comment]
No it worked the first time like everything was fine on the test server then I had to move it to the main server it started bugging so I deleted the files and installed everything again and that's when it all started breaking. But I mean it's all good is there another way I can save msg id of statusCH without sqlite? the service has MySQL. And I can't exactly switch services either I have always been with them and I have 2 other bots running currently on the service. If you are able to help me make the statusCH work another way that would be great if not it's all good I can figure something else out
